docs/bucket/policies/README.md

```diff
@@ -0,0 +1,110 @@
+# Bucket Access Management Quickstart Guide [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/)
+
+MinIO uses Policy-Based Access Control (PBAC) to define the authorized actions and resources to which an authenticated user has access. Each policy describes one or more `actions` and `conditions` that outline the permissions of a user or group of users.
+
```

Comment:
this is not a bucket policy instead its an IAM policy directly attached to the user

Reply:
ah ok. better rename the tests as `docs/iam/policies`. will do the needed changes.

Reply:
this documentation is not needed @shtripat

Reply:
> this documentation is not needed @shtripat

ok, will remove this. thanks!